docs/overview.rst

```diff
@@ -17,7 +17,9 @@ The original datasets, stored in the WCRP/CMIP6 ESGF repositories, consist of `n
 Each of these datasets typically corresponds to a single variable saved at specified time intervals for the length of a single model run.
 For convenience, these datasets were often divided into multiple netCDF files.
 Our Zarr data stores correspond to the result of concatenating these netCDF files and then storing them as a single Zarr object.
-All of the metadata in the original netCDF files has been preserved, including the licensing information and the CMIP6 persistent identifiers (``tracking_ids``) which are unique for each of the original netCDF files.
+All of the metadata in the original netCDF files has been preserved, including the licensing information and the CMIP6 persistent identifiers (``tracking_ids``) which are unique for each of the original netCDF files. 
+
+A Zarr data store ``tracking_id`` consists of a concatenated list of the netCDF ``tracking_ids`` from which it was created. An individual ``tracking_id`` can be looked up at `Handle.net <http://hdl.handle.net/>`_  (e.g., enter "hdl:21.14100/33cbdc29-fbc9-44ab-9e09-5dc7824441cf", which then redirects `here <https://handle-esgf.dkrz.de/lp/21.14100/33cbdc29-fbc9-44ab-9e09-5dc7824441cf/>`_).
```

Comment:
```suggestion
A Zarr data store ``tracking_id`` consists of a concatenated list of the netCDF ``tracking_ids`` from which it was created. An individual ``tracking_id`` can be looked up at `Handle.net <http://hdl.handle.net/>`_  (e.g., enter ``hdl:21.14100/33cbdc29-fbc9-44ab-9e09-5dc7824441cf``, which then redirects `here <https://handle-esgf.dkrz.de/lp/21.14100/33cbdc29-fbc9-44ab-9e09-5dc7824441cf/>`_).
```

Wasn't sure if this qualifies as a URL befitting monospaced font.

Reply:
@charlesbluca , thanks for taking a look!  I agree it would be handy to have a separate name for the Zarr `tracking_id` which is the concatenation of the netcdf `tracking_ids`.  ESGF already has an additional persistent identifier for what it considers a dataset (the collection of netcdf files in a given directory).  Unfortunately, they have already given something else the name `dataset_id` (e.g., 'dataset_id' = CMIP6.ScenarioMIP.NCAR.CESM2.ssp370.r4i1p1f1.Amon.ts.gn ) but don't seem to use it much.  

Since the Zarr tracking_id is not itself a persistent identifier, we should not really call it `zarr_tracking_id` or anything like that.  Maybe `netcdf_tracking_ids`?

Reply:
Oh yes, I like your 'suggested change', but don't know how to make it happen. Do I need to re-open the pull request and commit or do you need to do?

Reply:
I like the idea of `netcdf_tracking_ids` - I seem to remember having a conversation about this around the BigQuery schema, which also opts for that name for the collection of file tracking IDs.

As for making the change, it depends - if you try to edit the [overview.rst](https://github.com/pangeo-data/pangeo-cmip6-cloud/blob/master/docs/overview.rst) in this repo, does it say you can push changes directly to the file, or that you need to make a new branch and open a pull request?

If the former, then I'd say this is small enough where you can just push the changes directly, since it's pretty minimal. If the latter, then open up another pull request and I'll play around with your permissions so in the future you can commit directly to the repo.

Reply:
Actually, the tracking_id, "hdl:21.14100/33cbdc29-fbc9-44ab-9e09-5dc7824441cf", is not a URL - so I am not sure what fonts it should have - suggestions?

Reply:
I might still opt for monospaced, as currently it seems to get picked up as a URL on the site, though it doesn't link anywhere:

![image](https://user-images.githubusercontent.com/20627856/106637985-5a7f9880-6551-11eb-8d11-2b5a501d10ea.png)